projects/plugins/jetpack/modules/masterbar/admin-menu/admin-menu.js

```diff
@@ -1,6 +1,6 @@
 /* global ajaxurl, jetpackAdminMenu */
 
-( function () {
+( function ( $ ) {
```

Comment:
Do we need jQuery here?

We're using `.off` and `.width`, two methods that are easily replicable with vanilla JS:

- `.off`: https://youmightnotneedjquery.com/#off
- `.width`: https://youmightnotneedjquery.com/#get_width

I suppose we're already injecting jQuery somewhere on page load, so that's one thing we can ignore. But if we're not, then we can save some bytes.

Reply:
Thanks for the feedback. I did not really want to use Jquery but its already part of Core. The reason i need it here is because there are some custom event listeners added with `.on` One such example is the `click.wp-responsive` event (in core). To my knowledge, jquery has its own events registry which can't be manipulated through Vanilla JS.